libraries/botbuilder-runtime-integration-express/src/index.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as t from 'runtypes';
+import express from 'express';
+import { Configuration, getRuntimeServices } from 'botbuilder-runtime';
+import { IServices, ServiceCollection } from 'botbuilder-runtime-core';
+
+/**
+ * Options for runtime Express adapter
+ */
+const TypedOptions = t.Record({
+    /**
+     * Port that server should listen on
+     */
+    port: t.Number,
+
+    /**
+     * Path that the server will listen to for [Activities](xref:botframework-schema.Activity)
+     */
+    messagingEndpointPath: t.String,
+});
+
+export type Options = t.Static<typeof TypedOptions>;
+
+const defaultOptions: Options = {
+    messagingEndpointPath: '/api/messages',
+    port: 3978,
+};
+
+/**
+ * Start a bot using the runtime Express integration.
+ *
+ * @param applicationRoot application root directory
+ * @param settingsDirectory settings directory
+ * @param options options bag
+ */
+export async function start(
+    applicationRoot: string,
+    settingsDirectory: string,
+    options: Partial<Options> = defaultOptions
+): Promise<void> {
+    TypedOptions.check(options);
```

Comment:
This is subtle, but I would not default `options` in the parameter list (or default it to `{}`) and do `const resolvedOptions = TypedOptions.check(Object.assign({}, defaultOptions, options));` This way users only have to specify the keys they care to override and missing keys are defaulted.

Reply:
It is subtle, but the suggestion makes the implementation match the signature by truly accepting `Partial<Options>`. Good catch!